web/angular-wallet/src/app/app.component.ts

```diff
@@ -27,13 +24,11 @@ export class AppComponent implements OnInit, OnDestroy {
   isLoggedIn = false;
   selectedAccount: Account;
   accounts: Account[];
-  BLOCKCHAIN_STATUS_INTERVAL = 10000; // 10 secs
+  BLOCKCHAIN_STATUS_INTERVAL = 30000; // 30 secs
```

Comment:
Also reduced this to 30 secs, it was a bit spammy at 10 secs

Reply:
:+1